first-analysis-steps/davinci-grid.md

```diff
@@ -99,7 +101,9 @@ myApp = GaudiExec()
 myApp.directory = "./DaVinciDev_v44r6"
 j.application = myApp
 j.application.options = ['ntuple_options.py']
-j.application.readInputData('MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09b_Trig0x6138160F_Reco16_Turbo03_Stripping28NoPrescalingFlagged_ALLSTREAMS.DST.py')
+bkPath = '/MC/2016/Beam6500GeV-2016-MagDown-Nu1.6-25ns-Pythia8/Sim09c/Trig0x6138160F/Reco16/Turbo03/Stripping28r1NoPrescalingFlagged/27163002/ALLSTREAMS.DST'
+data  = BKQuery(bkPath,dqflag=['OK']).getDataset()
```

Comment:
There should be a space here to match with the Python style guide ([PEP8](https://www.python.org/dev/peps/pep-0008/)) and be consistent with the previous use:

```suggestion
data  = BKQuery(bkPath, dqflag=['OK']).getDataset()
```